arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+<map proto="1.4.2">
+<name>Bed Rush</name>
+<version>1.0.0</version>
+<phase>development</phase>
+<objective>Destroy the Bed to win the Round.  First Team to 5 Wins!</objective>
```

Comment:
Accidental double spacing between these two sentences.

Reply:
Not accidental, but I can update